tests/unit/mathematics/Statistics_unittest.cc

```diff
@@ -539,3 +540,89 @@ TEST(Statistics, vector_mean_overflow_test)
 	a.set_const(std::numeric_limits<float64_t>::max());
 	EXPECT_EQ(std::numeric_limits<float64_t>::max(), CStatistics::mean(a));
 }
+
+TEST(Statistics, variance_test)
+{
+	Eigen::VectorXd a(5);
+	a << 600, 470, 170, 430, 300;
+	EXPECT_EQ(linalg::variance(a), 21704);
+}
+
+TEST(Statistics, matrix_variance_test_colwise)
```

Comment:
Let's use `col_wise` and `row_wise` for the test names for consistency with the implementation above.